fuel-relayer/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "fuel-relayer"
+version = "0.0.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+repository = "https://github.com/FuelLabs/fuel-core"
+description = "Fuel Relayer"
+publish = false
+
+[dependencies]
+anyhow = "1.0"
+async-trait = "0.1"
+bytes = "1.1"
+env_logger = "0.9"
+ethers-core = { git = "https://github.com/rakita/ethers-rs.git", branch = "master", package = "ethers-core", default-features = false }
```

Comment:
Can we not use the versions available from crates.io?

Reply:
Ref: https://github.com/FuelLabs/fuel-core/pull/207#discussion_r826727630

Reply:
I added few things in ethers that is in current master, when it is published we will switch to cargo version

Reply:
wow, for some reason I didn't notice John's comment right below saying the same thing. Sorry, for the dupe.